onebusaway-android/src/main/res/values/themes.xml

```diff
@@ -25,5 +25,14 @@
         <item name="autoCompleteTextViewStyle">@style/cursorColor</item>
         <!-- Allows us to force Dark Theme throughout the app -->
         <item name="android:forceDarkAllowed">true</item>
+
     </style>
+
+    <!-- Custom TimePickerDialog theme for dark mode -->
+    <style name="MyTimePickerDialogTheme" parent="Theme.MaterialComponents.DayNight.Dialog">
```

Comment:
I think the name of this style should be more clear about its intent than `MyTimePickerDialogTheme`. Maybe `dayNightTimePickerDialogTheme` instead?